movienager/accounts/management/commands/_user_factory.py

```diff
@@ -0,0 +1,14 @@
+import factory
```

Comment:
I would create `factory` module in the `accounts` package root - factories may be even more useful in tests, so they can be more visible in the code base